src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/InputSeparatorWrapSetOptionTrim.java

```diff
@@ -0,0 +1,21 @@
+/*
+SeparatorWrap
+option = \tNL
+tokens = METHOD_REF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.separatorwrap;
+
+import java.util.Arrays;
+
+public class InputSeparatorWrapSetOptionTrim {
+
+    void Method() {
+        String[] stringArray = { "Barbara", "James", "Mary", "John",
+            "Patricia", "Linda" }; // violation below ''::' should be on a new line'
+        /* violation */ Arrays.sort(stringArray, String::
```

Comment:
As you create new file.
Please make it with indentation of +2, not a +4.
Remove this leading comment.
Move trailing comment to same line where violation